CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## 1.23.0 (Unreleased)
 
+## 1.22.4 (January 20, 2021)
+
+FEATURES:
+* **New Data Source:** `opentelekomcloud_rds_versions_v3` ([#792](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/792))
+
+ENHANCEMENTS:
+* `resource/opentelekomcloud_cce_cluster_v3`: Implement plan stage network validation ([#787](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/787))
```

Comment:
sort changes please

Reply:
#792 is in `FEATURES` section, inside of section entities are already sorted